scripts/docker_build.sh

```diff
@@ -17,8 +18,17 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
+# Docker from ORT uses BuildKit feature
+DOCKER_BUILDKIT=1
+export DOCKER_BUILDKIT
```

Comment:
FYI, this is now also in upstream; you'll see when you rebase.

Reply:
Done on last commit